client/lib/plans/constants.js

```diff
@@ -631,6 +649,26 @@ export const FEATURES_LIST = {
 		getTitle: () => 'Unlimited Backup Space', //PLANS A/B TEST: Translate if test passes
 	},
 
+	[ FEATURE_FREE_WORDPRESS_THEMES ]: {
+		getSlug: () => FEATURE_FREE_WORDPRESS_THEMES,
```

Comment:
Any good reason to not wrap all of these in a `translate` call immediately? In the worst case we could lose several translated versions of it if we decide to not keep the test.